frontend/packages/dev-console/src/components/topology/topology-types.ts

```diff
@@ -60,7 +61,7 @@ export interface TopologyDataObject<D = {}> {
   id: string;
   name: string;
   type: string;
-  resources: ResourceProps[];
+  resources: any;
```

Comment:
Downgrading a prop to `any` isn't good. What is the structure of the object? Key-map to what value?

More on map types in TypeScript: http://www.typescriptlang.org/docs/handbook/advanced-types.html#mapped-types

Reply:
@sahil143 this needs to be addressed before PR can be merged.

Reply:
fixed in https://github.com/openshift/console/pull/2849/commits/116a1f08ce828a61035940e67d4991ea8918da7b